src/Sylius/Bundle/PaymentBundle/Resources/config/services/provider.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Sylius Sp. z o.o.
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<container
+    xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd"
+>
+    <services>
+        <defaults public="true" />
+
+        <service id="sylius.payment_request.provider.gateway_factory_name" class="Sylius\Bundle\PaymentBundle\Provider\GatewayFactoryNameProvider" />
```

Comment:
Because this service uses payment request and this is consistent with other services added for payment request functionality, so I leave this part in id. However, now I'm wondering if we shouldn't add a separate directory in PaymentBundle for payment requests 🤔